app/entities/user.go

```diff
@@ -87,3 +94,11 @@ func checkPass(password string) error {
 
 	return nil
 }
+
+func (u UserID) ValidateUUID() error {
+	if valid := IDRegex.MatchString(u.ID); !valid {
```

Comment:
Please take a look at this approach - uuid.Parse().
You can try to parse it and if it doesn't return an error - it's a valid.
It should be much faster than regexp.
On top of that, it's easier to read and support that regexp.

Reply:
Thanks for the tip. Really it's better